<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookmarksDialog</class>
 <widget class="QDialog" name="BookmarksDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bookmarks</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListView" name="bookmarkList">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="chooseButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="sourceButton">
     <property name="text">
      <string>Add Departure</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="targetButton">
     <property name="text">
      <string>Add Destination</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="deleteButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
